src/io/flutter/module/FlutterGeneratorPeer.form

```diff
@@ -82,6 +74,28 @@
           </scrollpane>
         </children>
       </grid>
+      <grid id="bdced" layout-manager="GridLayoutManager" row-count="1" column-count="1" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+        <margin top="0" left="0" bottom="0" right="32"/>
+        <constraints>
+          <grid row="1" column="2" row-span="1" col-span="1" vsize-policy="3" hsize-policy="3" anchor="4" fill="2" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties/>
+        <border type="none"/>
+        <children>
+          <component id="5f26c" class="com.intellij.ui.components.labels.LinkLabel" binding="myInstallActionLink">
+            <constraints>
+              <grid row="0" column="0" row-span="1" col-span="1" vsize-policy="3" hsize-policy="3" anchor="0" fill="1" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <horizontalAlignment value="0"/>
+              <horizontalTextPosition value="4"/>
+              <iconTextGap value="0"/>
+              <text value="Install..."/>
```

Comment:
We should probably indicate that this will show the user some install instructions. Perhaps, `'View SDK install instructions...'`? `View setup docs...`?